### DISCUSSION TITLE
why not prefer jsoncpp or json-c? for json usage?

### DISCUSSION THREAD

**Discussion by sewnie** [uowuo/abaddon#132] [General]
https://github.com/open-source-parsers/jsoncpp
https://github.com/json-c/json-c

**Comment by ouwou** [comment]
is there a reason i should? nlohmann/json works fine and i dont see any reason to go through the trouble of switching

**Reply by sewnie** [reply]
i'm not exactly saying to suggest switching; but rather why prefer nlohmann's json c++ implementations at first?

**Reply by sewnie** [reply]
well, especially if there are simpler and better alternatives

**Reply by ouwou** [reply]
ive used it before, its simple, its fast, its lightweight, it does what i want, etc

**Reply by sewnie** [reply]
thanks for your answer.
